components/eamxx/src/physics/ace/eamxx_ace_process_interface.cpp

```diff
@@ -0,0 +1,855 @@
+    // Sea surface temperature [K]
+    add_field<Computed>("sst",              scalar2d, K,       grid_name);
+    //dust fluxes [kg/m^2/s]: Four flux values for eacch column
+    add_field<Computed>("dstflx",           vector4d, kg/m2/s, grid_name);
+
+    ----------------------------
+    ACE emulator requires these:
+    ----------------------------
+    - land_fraction
+    - ocean_fraction
+    - sea_ice_fraction
+    - DSWRFtoa # solin
+    - HGTsfc
+    - PRESsfc
+    - surface_temperature
+    - air_temperature_0 # _0 denotes the top most layer of the atmosphere
+    - air_temperature_1
+    - air_temperature_2
+    - air_temperature_3
+    - air_temperature_4
+    - air_temperature_5
+    - air_temperature_6
+    - air_temperature_7
+    - specific_total_water_0
+    - specific_total_water_1
+    - specific_total_water_2
+    - specific_total_water_3
+    - specific_total_water_4
+    - specific_total_water_5
+    - specific_total_water_6
+    - specific_total_water_7
+    - eastward_wind_0
+    - eastward_wind_1
+    - eastward_wind_2
+    - eastward_wind_3
+    - eastward_wind_4
+    - eastward_wind_5
+    - eastward_wind_6
+    - eastward_wind_7
+    - northward_wind_0
+    - northward_wind_1
+    - northward_wind_2
+    - northward_wind_3
+    - northward_wind_4
+    - northward_wind_5
+    - northward_wind_6
+    - northward_wind_7
+  */
+
+  using KT  = KokkosTypes<DefaultDevice>;
+  using MT  = typename KT::MemberType;
+  using ESU = ekat::ExeSpaceUtils<typename KT::ExeSpace>;
+
+  auto input_view = m_input_field.get_view<Real ****>();
+
+  auto land_fraction = m_land_fraction_ll.get_view<const Real **>();
+  auto ocean_fraction = m_ocean_fraction_ll.get_view<const Real **>();
+  auto sea_ice_fraction = m_sea_ice_fraction_ll.get_view<const Real **>();
+  auto surf_radiative_T = m_surf_radiative_T_ll.get_view<const Real **>();
+  auto T_mid = m_T_mid_ll.get_view<const Real ***>();
+  auto qv = m_qv_ll.get_view<const Real ***>();
+  auto wind = m_wind_ll.get_view<const Real ****>();
+  auto p_int = m_p_int_ll.get_view<const Real ***>();
+
+  const auto num_lat = m_height;
+  const auto policy = ESU::get_default_team_policy(m_width, m_height);
+
+  // Populate the view of input data
+  // TODO: bypass this and just fill in the tensor directly?
+  // ALSO: note hacks!
+  Kokkos::parallel_for(
+      "ACE::preprocess", policy, KOKKOS_LAMBDA(const MT &team) {
+        const int ilon = team.league_rank();
+        Kokkos::parallel_for(
+            Kokkos::TeamVectorRange(team, num_lat), [&](const int &ilat) {
+              input_view(0, 0, ilat, ilon) = land_fraction(ilat, ilon);
+              input_view(0, 1, ilat, ilon) = ocean_fraction(ilat, ilon);
+              input_view(0, 2, ilat, ilon) = sea_ice_fraction(ilat, ilon);
+              // DSWRFtoa (SOLIN): TODD (BEN???)
+              input_view(0, 3, ilat, ilon) = 500.0;
+              // HGTsfc (surface height): TODO (topo file?)
```

Comment:
Yes, topography can presumably come from the topo file. For the E3SM checkpoiont, that's where we get the data anyways